new/LIT7052Qene.xml

```diff
@@ -0,0 +1,76 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7052Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ኢሀሎ፡ በዓለም፡ ዲበ፡ ሰረገላ፡ እንዘ፡ በዘባኑ፡ ለአድግ፡ ነባቤ፡ ነገር፡ ሠናይ።</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾI-hallo ba-ʿālam diba saragallā ʾǝnza ba-zabānu la-ʾadg nabābe nagar 
+                    śannāy. (Qǝne of the Kʷǝllǝkǝmu gǝʿǝz-type)</title>
+                <title xml:lang="en" corresp="#t1">In the world, he is not on the chariot, but on the back of the donkey, the speaker of the good 
+                    things. ...</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="BNFabb145"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Qǝne of the Kʷǝllǝkǝmu gǝʿǝz-type. Type and author are not indicated in the manuscript.</p>
```

Comment:
I would transcribe "gǝʿz", as we are not talking about the language here

Reply:
It is transcribed like this in Bm. I want to keep consistency in the transcription: 

<img width="128" alt="Screenshot 2024_Kwellekemu geez" src="https://github.com/user-attachments/assets/bfcddd7b-b082-47cf-8c77-7bd4c8bdbdaa">

Reply:
If that case, I think it would be better to change the name of the qene form and use "gǝʿz" everywhere